book/src/pong/09_keeping_score.md

```diff
@@ -30,7 +30,7 @@ Firstly, let's add the score to the `Paddle` objects:
 ```rust
 struct Paddle {
     pos: Vector2D<Fixed>,
-    health: i32,
```

Comment:
do you think we should add a `get_health(&self)` method to Paddle instead of making this `pub`? (legit question, I'm not sure what the best answer here is :grin:)

Reply:
I think `get_health()` and `decrement_health()` is much nicer. Makes it clear what we expect you to do with the health field. Good suggestion!